index.js

```diff
@@ -1,86 +0,0 @@
-//Sorry about the multiple files. This will definitely be one I'll re-submit...
-//the ones to look at are (under the guess-the-number folder) are: extended range.js, index.js, and inversegame.js
-//I know it's mostly there but tweaks definitely need to be made.
-//Couldn't figure out how to combine the two versions, had difficulty with the cheat clauses
-//and something in the randNum function in the extended range version is off
-//I just could not finish it the way I wanted to for the life of me
-
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-function randNum(min, max) {
-  let range = (max - min) + 1;
-  let decimal = (Math.random() * range);
-  let int = Math.floor(decimal) + min;
-  return int;
-}
-
-
-
-async function start() {
-
-
-
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("Pick a number between 1 and 100 - What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-
```

Comment:
In this line you can write something similar to line 42 saying something that would check to see if you secret number is actually a number